src/lollipopChart.js

```diff
@@ -362,7 +388,7 @@ var LollipopChart = function (selection) {
   chart.lollipopRadius = function(_) {
     if(!arguments.length) return lollipopRadius;
     lollipopRadius = _;
-    chartGutter = _;
+    //chartGutter = _; //is this a bug?
```

Comment:
Ah so the gutter gets calculated correctly in the data function? I guess it just needs to get calculated correctly in the lollipopRadius function too.